starter_templates/zig/code/.codecrafters/run.sh

```
@@ -8,4 +8,4 @@
 
 set -e # Exit on failure
 
-exec $(dirname $0)/zig-out/bin/zit "$@"
+exec $(dirname $0)/zig-out/bin/main "$@"
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)